ckanext/validation/plugin.py

```diff
@@ -51,16 +50,6 @@ def i18n_directory(self):
     # IConfigurer
 
     def update_config(self, config_):
-        if not tables_exist():
```

Comment:
Can leave in. Worked out how to test of tables exist in my fork

Reply:
We don't really need it, though. `init_tables` checks if things exist.